collateral-info.yaml

```diff
@@ -215,6 +215,10 @@ collateral-defs:
     collateral-shortname: 6s Capital
     collateral-symbol: RWA001
     collateral-info-url: https://manual.makerdao.com/collateral/real-world-assets-in-detail/6s
+    collateral-stability-fee-type: Fixed
+    collateral-stability-fee-payment-frequency: By the second
+    collateral-documentation: https://ipfs.io/ipfs/QmdmAUTU3sd9VkdfTZNQM6krc9jsKgF2pz7W1qvvfJo1xk
```

Comment:
So I've previously used 'collateral-info-url' to link to docs, but I worry the difference between that and 'collateral-documentation' are not clear. I would suggest 'collateral-rwa-docs' to disambiguate.

Reply:
I've decided to remove this  links. These links are available on-chain. Adding them here will require manual update. The metadata is for info that is not on-chain and needs to be updated manually, rarely, so adding this IPFS links makes no sense now that I think about it.